src/main/java/games/strategy/engine/pbem/PBEMMessagePoster.java

```diff
@@ -150,7 +150,8 @@ public boolean post(final IDelegateHistoryWriter historyWriter, final String tit
   }
 
   /**
-   * Converts text to html, by transforming \n to <br/>.
+   * Converts text to html, by transforming \n to <br/>
+   * .
```

Comment:
I think the formatter is forcing the period onto a new line because this Javadoc is technically malformed.  Since the `javadoc` tool processes tags, the `<br/>` in this Javadoc should probably be written as `&lt;br/>` (you can observe the change in behavior in the Eclipse Javadoc view).

Reply:
You probably mean `&lt;br/&gt;` ^^